packages/contracts-bedrock/scripts/deploy/UpgradeAnchorStateRegistry.s.sol

```diff
@@ -18,67 +18,61 @@ import { IDisputeGameFactory } from "interfaces/dispute/IDisputeGameFactory.sol"
 import { IAnchorStateRegistry } from "interfaces/dispute/IAnchorStateRegistry.sol";
```

Comment:
Duplicated imports of IAnchorStateRegistry?

Reply:
Fixed.